tests/acceptance/test_issue_details.py

```diff
@@ -79,11 +79,24 @@ def test_cocoa_event(self):
         self.page.visit_issue(self.org.slug, event.group.id)
         self.browser.snapshot("issue details cocoa")
 
+    def test_cocoa_event_breadcrumb_v2(self):
+        with self.feature("organizations:breadcrumbs-v2"):
+            # TODO: This should become its own "rust" platform type
+            event = self.create_sample_event(platform="native", sample_name="Rust")
```

Comment:
Should this be cocoa?

Reply:
sure thing :) I will fix in another PR ...promise 🤞